### PR TITLE
Export personal note and comments in PDFs

### PR DESCRIPTION
**Pull request by FinnStutzenstein**

### CODE REVIEW

**Comment by FinnStutzenstein** [comment]
@normanjaeckel @emanuelschuetze Test again

File: openslides/motions/static/templates/motions/motion-detail/personal-note.html [hunk 1]
```diff
@@ -14,6 +14,12 @@
           <translate>Inline editing</translate>
         </button>
       </div>
+      <div class="pull-right inline-editing-activator">
+        <button ng-click="exportPersonalNote()" class="btn btn-default btn-sm">
+          <i class="fa fa-file-pdf-o"></i>
+          <translate>Export personal note</translate>
```
Comment (emanuelschuetze) [inline openslides/motions/static/templates/motions/motion-detail/personal-note.html]: Please use just "PDF" as button text (like the motion pdf button) and add these string ("Export personal note") as tooltip.

**Comment by FinnStutzenstein** [comment]
@emanuelschuetze Comments can be exported now

**Comment by FinnStutzenstein** [comment]
@emanuelschuetze Ok.

**Comment by emanuelschuetze** [comment]
Rebased and CHANGELOG entry added.

**Pull request merged**